javascriptv3/example_code/lambda/scenarios/basic/README.md

```diff
@@ -0,0 +1,34 @@
+# Scenario - AWS Lambda for AWS SDK for JavaScript (v3)
+
+This is an interactive command prompt that showcases how the JavaScript SDK can be
+used for interacting with Lambda.
+
+For the purpose of this example `root` refers to the same directory as this readme.
+
+## Prerequisites
+
+- create an AWS account
+- setup a [credentials file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html)
+- configure your account (or IAM user) with access to Lambda and IAM
+- install `nodejs` version 18.7.0
+- install `yarn`
+
+## Running the example
+
+### Install dependencies for this example
+
+The included setup script installs dependencies for this example,
+the included Lambda functions, and a few IAM commands.
+
+1. Run `yarn`
+
+### Service usage warning
+
+Running the below commands may cause charges to your AWS account.
+Make sure you understand billing model for the services used here.
```

Comment:
You could include a link to the pricing page here: https://aws.amazon.com/pricing